KamerConnect/Models/HousePreferences.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace KamerConnect.Models
+{
+    public class HousePreferences
+    {
+        public string Budget;
+        public string SurfaceArea;
+        public HouseType Type;
+
+        public HousePreferences(string budget, string surfaceArea, HouseType type)
+        {
+            Budget = budget;
```

Comment:
Zijn dit echt alle preference velden? Als dit niet zo is maak er maar een losse ticket voor aan dan doen we dat na de demo.
Ik mis ook een stukje locatie namelijk.